botbot_plugins/plugins/jira.py

```diff
@@ -0,0 +1,69 @@
+import requests
+import json
+from urlparse import urljoin
+from .. import config
+from ..base import BasePlugin
+from ..decorators import listens_to_all, listens_to_mentions
+
+
+
+class Config(config.BaseConfig):
+    jira_link = config.Field(help_text="Jira Link, eg: 'https://tickets.metabrainz.org'", default="https://tickets.metabrainz.org")
```

Comment:
I think it would be better to change the help text to "Jira URL" and not provide a default value. That way we can easily PR this upstream to @BotBotMe.